plugins/coverbrowser.koplugin/main.lua

```diff
@@ -192,42 +190,70 @@ function CoverBrowser:addToMainMenu(menu_items)
                     UIManager:show(items)
                 end,
             },
+            {
+                text = _("Length / progress"),
```

Comment:
We generally prefer to phrase as `Length and progress` if I'm reading this correctly.